src/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -686,12 +686,6 @@ smt_termt convert_expr_to_smt(const exprt &expr)
   {
     return convert_expr_to_smt(*float_not_equal);
   }
-  if(
```

Comment:
⛏️ Can you keep commit titles short enough that they display in their entirety in the github UI without having to expand them?

Reply:
This looks to be resolved.